buildroot/share/PlatformIO/scripts/schema.py

```diff
@@ -121,7 +121,12 @@ class Parse:
     # Pattern to match a float value
     flt = r'[-+]?\s*(\d+\.|\d*\.\d+)([eE][-+]?\d+)?[fF]?'
     # Defines to ignore
-    ignore = ('CONFIGURATION_H_VERSION', 'CONFIGURATION_ADV_H_VERSION', 'CONFIG_EXAMPLES_DIR', 'CONFIG_EXPORT')
+    ignore = (
+        "CONFIGURATION_H_VERSION",
+        "CONFIGURATION_ADV_H_VERSION",
+        "CONFIG_EXAMPLES_DIR",
+        "CONFIG_EXPORT",
+    )
```

Comment:
this seems like inconsistent formatting
You join separate import lines into one line
but then you split up lists on one line  into separate lines.

Reply:
this can be removed `ignore` is unused